src/music.cpp

```diff
@@ -0,0 +1,106 @@
+#include "music.h"
+#include "debug.h"
+
+namespace music
+{
+
+std::vector<std::pair<music_id, bool>> music_id_list { std::make_pair( mp3, false ), std::make_pair( instrument, false ), std::make_pair( sound, false ), std::make_pair( title, true ) };
+
+bool is_active_music_id( music_id data )
+{
+    for( std::pair<music_id, bool> &pair : music_id_list ) {
+        if( pair.first == data ) {
+            return pair.second;
+        }
+    }
+
+    return false;
+}
+
+std::string enum_to_string( music_id data )
+{
+    switch( data ) {
+        case mp3:
+            return "mp3";
+        case instrument:
+            return "instrument";
+        case sound:
+            return "sound";
+        case title:
+            return "title";
+    }
+
+    cata_fatal( "Invalid music_id in music::enum_to_string" );
+}
+
+music_id string_to_enum( std::string data )
+{
+    for( std::pair<music_id, bool> &pair : music_id_list ) {
+        if( !data.compare( enum_to_string( pair.first ) ) ) {
```

Comment:
```suggestion
        if( data == enum_to_string( pair.first ) ) {
```
If you decide not to go with anothersim's suggestion (using a `io::` template), you should at least use the comparison operator to compare strings

Reply:
I used io:: template based on anothersim's suggestion.